subxt/src/config.rs

```diff
@@ -78,7 +78,14 @@ pub trait Config: 'static {
     type Signature: Verify + Encode + Send + Sync + 'static;
 
     /// Extrinsic type within blocks.
-    type Extrinsic: Parameter + Extrinsic + Debug + MaybeSerializeDeserialize;
+    // TODO(niklasad1): I have no idea if this ok or not ^^ => won't work otherwise
+    // because `jsonrpsee` requires `Deserialize + Send + Sync` for the return types.
```

Comment:
Andrew would know better, but afaiu this is pretty much going to be some wrapped `Vec<u8>` I'd expect, so I don't see any reason why Send + Sync would be an issue if required!